admin_console/forms/PartnerConfiguration.php

```diff
@@ -477,6 +477,11 @@ public function init()
 			'decorators' => array('ViewHelper', array('HtmlTag',  array('tag' => 'dt', 'id' => 'mothly_bandwidth_combined')))
 		));
 		
+		$this->addElement('text', 'partner_level_allowed_domains', array(
```

Comment:
can be just allowed_domaind as the whole configuration is partner configuration
also move it to the section where we have 'CDN Host white list regex' as this new option is not related to 'Included Usage' section